packages/stylelint-plugin/src/kaizenTokens.ts

```diff
@@ -0,0 +1,160 @@
+import fs from "fs"
+import flatmap from "lodash.flatmap"
+import postcssValueParser from "postcss-value-parser"
+import { CSSVariable, KaizenToken } from "./types"
+import { getParser } from "./utils"
+import { isVariable } from "./variableUtils"
+const scssParser = getParser("scss")
+
+/* Pass in just the name of a module which is used to import variable.
+  E.g. "color" or "color-vars", NOT "@kaizen/design-tokens/sass/color".
+  Assumes that the SASS and LESS modules contain the same variables
+*/
+const getVarsFromKaizenModule = (moduleName: string) => {
+  const sassModulePath = `@kaizen/design-tokens/sass/${moduleName}`
```

Comment:
Potentially import the TS files from design-tokens since that is treated as the source of truth for token generation.